### ISSUE TITLE
Tar with pre-trained models corrupted

### ISSUE THREAD

**Issue by IvonaTau** [whai362/PSENet#74]
Hi I tried all the pretrained models from OneDrive but it is not possible to extract them. The .tar files are corrupted.

**Comment by ltm920716** [comment]
@IvonaTau change the suffix directly

**Comment by azad96** [comment]
@IvonaTau you should use the pretrained models directly without extracting.

**Comment by PaulCreaserML** [comment]
Yep, they are not actually tar files. You can simply use as is, or if you wish rename them, but without the tar extension.
